snap-core/pom.xml

```diff
@@ -246,6 +247,11 @@
             <version>3.9.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.googlecode.json-simple</groupId>
+            <artifactId>json-simple</artifactId>
+        </dependency>
+
```

Comment:
great. This is getting commonly used. Having it in core we can remove it as a public package in a few other places.